scripts/base/protocols/rdp/consts.zeek

```diff
@@ -97,42 +97,65 @@ export {
 		[6145] = "Arabic - Morocco",
 		[8193] = "Arabic - Oman",
 		[16385] = "Arabic - Qatar",
+		[1025] = "Arabic - Saudi Arabia",
 		[10241] = "Arabic - Syria",
 		[7169] = "Arabic - Tunisia",
 		[14337] = "Arabic - U.A.E.",
 		[9217] = "Arabic - Yemen",
+		[132097] = "Arabic (102) AZERTY",
+		[66561] = "Arabic (102)",
 		[1067] = "Armenian - Armenia",
+		[132139] = "Armenian Phonetic",
+		[197675] = "Armenian Typewriter",
+		[66603] = "Armenian Western",
 		[1101] = "Assamese",
+		[66604] = "Azerbaijani (Standard)",
 		[2092] = "Azeri (Cyrillic)",
 		[1068] = "Azeri (Latin)",
+		[66629] = "Bangla (India - Legacy)",
+		[132165] = "Bangla (India)",
 		[1133] = "Bashkir",
 		[1069] = "Basque",
 		[1059] = "Belarusian",
-		[1093] = "Bengali (India)",
+		[67596] = "Belgian (Comma)",
 		[2117] = "Bengali (Bangladesh)",
+		[1093] = "Bengali (India)",
 		[5146] = "Bosnian (Bosnia/Herzegovina)",
+		[8218] = "Bosnian (Cyrillic) - Bosnia and Herzegovina",
 		[1150] = "Breton",
+		[723968] = "Buginese",
+		[66562] = "Bulgarian (Latin)",
+		[132098] = "Bulgarian (phonetic layout)",
+		[263170] = "Bulgarian (phonetic traditional)",
 		[1026] = "Bulgarian",
+		[197634] = "Bulgarian",
 		[1109] = "Burmese",
+		[69641] = "Canadian Multilingual Standard",
 		[1027] = "Catalan",
+		[4191] = "Central Atlas Tamazight (Tifinagh) - Morocco",
 		[1116] = "Cherokee - United States",
+		[66652] = "Cherokee Nation Phonetic",
+		[3076] = "Chinese - Hong Kong SAR",
+		[5124] = "Chinese - Macao SAR",
 		[2052] = "Chinese - People's Republic of China",
 		[4100] = "Chinese - Singapore",
 		[1028] = "Chinese - Taiwan",
-		[3076] = "Chinese - Hong Kong SAR",
-		[5124] = "Chinese - Macao SAR",
+		[31748] = "Chinese - Traditional",
 		[1155] = "Corsican",
-		[1050] = "Croatian",
 		[4122] = "Croatian (Bosnia/Herzegovina)",
+		[4122] = "Croatian (Latin) - Bosnia and Herzegovina",
```

Comment:
This one looks duplicated from the one above it.

Reply:
I blame Notepad as my IDE :)